Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -495,15 +537,15 @@ public IEnumerable<IMixedRealityManager> GetManagers(Type type)
         /// <returns>An array of Managers that meet the search criteria</returns>
         public List<IMixedRealityManager> GetManagers(Type type, string managerName)
         {
-            if (!ActiveProfile)
+            if (ActiveProfile == null)
             {
                 Debug.LogError($"Unable to add a new {type.Name} Manager as the Mixed Reality manager has to Active Profile");
                 return null;
             }
 
             if (type == null) { throw new ArgumentNullException(nameof(type)); }
 
-            List<IMixedRealityManager> managers = new List<IMixedRealityManager>();
+            var managers = new List<IMixedRealityManager>();
```

Comment:
Why change this to a var?  It's a fixed implementation with a concrete type, a var is just going to add unnecessary overhead

Reply:
syntactic sugar I suppose.

Reply:
I usually go with var if the type is explicit